frontend/src/components/eMIB/TestInstructions.jsx

```diff
@@ -36,7 +36,7 @@ class TestInstructions extends Component {
     const exampleEmailResponse = {
       actionType: ACTION_TYPE.email,
       emailType: EMAIL_TYPE.reply,
-      emailTo: [8], // Geneviève Bédard in the address book
+      emailTo: [{ name: "Geneviève Bédard (Research and Innovations Director)", id: 8 }],
```

Comment:
I don't understand here why we have an hard coded name? Will that be translated?

Reply:
Agreed; I don't think this will be translated.

Reply:
You're right - I'll refactor this.